README.md

```diff
@@ -1,48 +1,68 @@
 # LSP
 
-[![GitHub release](https://img.shields.io/github/release/tomv564/LSP.svg)](https://github.com/tomv564/LSP/releases) [![Build Status](https://travis-ci.org/tomv564/LSP.svg?branch=master)](https://travis-ci.org/tomv564/LSP) [![Coverage Status](https://coveralls.io/repos/github/tomv564/LSP/badge.svg?branch=master)](https://coveralls.io/github/tomv564/LSP?branch=master) [![license](https://img.shields.io/github/license/mashape/apistatus.svg)]() [![Join the chat at https://gitter.im/SublimeLSP/Lobby](https://badges.gitter.im/SublimeLSP/Lobby.svg)](https://gitter.im/SublimeLSP/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![GitHub release](https://img.shields.io/github/release/tomv564/LSP.svg)](https://github.com/tomv564/LSP/releases) 
+[![Build Status](https://travis-ci.org/tomv564/LSP.svg?branch=master)](https://travis-ci.org/tomv564/LSP) 
+[![Coverage Status](https://coveralls.io/repos/github/tomv564/LSP/badge.svg?branch=master)](https://coveralls.io/github/tomv564/LSP?branch=master) 
+[![Join the chat at https://gitter.im/SublimeLSP/Lobby](https://badges.gitter.im/SublimeLSP/Lobby.svg)](https://gitter.im/SublimeLSP/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Gives Sublime Text 3 rich editing features for languages with Language Server Protocol support.
+Language Server Protocol support for Sublime Text 3 that gives you IDE features.
 
-Tested against language servers for javascript, typescript, python, php, java, go, c/c++ (clangd), scala (dotty), julia, rust, reason.
+![diagnostics screen-shot](docs/images/showcase.gif "TypeScript Server Example")
 
-See [Langauge Server Protocol](https://microsoft.github.io/language-server-protocol/) for available implementations.
+## Installation
 
-## Features
+### Stable Version
 
-Completions with snippet support.
+* Open the command palette and run `Package Control: Install Package` then select `LSP`.
 
-Navigate code with `Go to Symbol Definition` and `Find Symbol References`
 
-Inline documentation from Hover and Signature Help popups
+### Development Version
 
-![hover screenshot](https://raw.githubusercontent.com/tomv564/LSP/master/docs/images/screenshot-hover.png)
+* Clone this repository into your Packages directory.
+* Open the command palette and run `Package Control: Satisfy Dependencies`.
 
-As-you-type diagnostics with support for code fixes (`F4` to select, `super+.` to trigger actions)
 
-![diagnostics screenshot](https://raw.githubusercontent.com/tomv564/LSP/master/docs/images/screenshot-diagnostics-action.png)
+## Getting started
 
-## Installing
+Copy the configuration for a <a href="#available_languages">language</a>. 
```

Comment:
I've been adding configs in the default_clients to avoid this copy/paste business.
In most cases, the steps should be something like:

# install server to its default location on the PATH.
# enable config in LSP
# open document

We could then point to the language docs only if a more custom setup is needed or desired.